facefusion/face_analyser.py

```diff
@@ -132,15 +132,30 @@ def detect_with_retinaface(temp_frame : Frame, temp_frame_height : int, temp_fra
 	feature_strides = [ 8, 16, 32 ]
 	feature_map_channel = 3
 	anchor_total = 2
-	prepare_frame = numpy.zeros((face_detector_height, face_detector_width, 3))
-	prepare_frame[:temp_frame_height, :temp_frame_width, :] = temp_frame
-	temp_frame = (prepare_frame - 127.5) / 128.0
+	offset_width = (face_detector_width - temp_frame_width) / 2
+	offset_height = (face_detector_height - temp_frame_height) / 2
+	reduction_size = round(min(face_detector_height, face_detector_width) / 2)
+	reduction_ratio_width = face_detector_width / reduction_size
+	reduction_ratio_height = face_detector_height / reduction_size
+	reduction_offset_width = (face_detector_width - reduction_size) / 2
+	reduction_offset_height = (face_detector_height - reduction_size) / 2
+	temp_frame = cv2.copyMakeBorder(temp_frame, round(offset_height - 0.1), round(offset_height + 0.1), round(offset_width - 0.1), round(offset_width + 0.1), cv2.BORDER_CONSTANT, value = (114, 114, 114))
+	temp_frame_redused = temp_frame.copy()
```

Comment:
temp_frame_reduced <- do not use adjectives and try to keep 2 step naming like prepare_frame